lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -1332,23 +1332,10 @@ void __cdecl _alloca_probe_16()
 #endif
         Js::Var varResult;
 
-        //The ARM can pass 4 arguments via registers so handle the cases for 0 or 1 values without resorting to asm code
```

Comment:
Can we limit these changes to non WIN32? (including the one above?)

Reply:
Reverted those as you requested (updated [reviewable](https://reviewable.io/reviews/Microsoft/ChakraCore/4452#-)), but they're tied to the hand-written assembly function. As I'm not sure if you meant that you're using the same files for another project or something else, I didn't do changes to the assembly - let me know if you want the old version compiled for Win32.

I actually guarded most of the changes so they're compiled for `__ANDROID__` only, but the .asm function seemed broken and I modified it for all platforms.

Reply:
I was only referring to this file (JavascriptFunction.cpp)

Reply:
All right, that's done then.